assets/proposals/0245-sharing-wifi-ssid-and-password/mobile_access_point_multidev_externalhost.puml

```diff
@@ -0,0 +1,129 @@
+participant HMI
+participant SDL
+box "Device1"
+participant APP1
+end box
+box "Device2"
+participant APP2
+end box
+
+note over HMI, APP2: in ini file : NetworkHost = MOBILE \n\
+Apps allowed to use feature by policies
+
+SDL -> HMI : <b>UI.GetCapabilities() request
+HMI -> SDL : <b>UI.GetCapabilities(NetworkingCapabilities) response : \n\
+\t autoJoinWiFiSupported : \t true \n\
+\t canHostWiFiNetwork : \t true \n\
+\t preferredNetworkHost : \t **MOBILE** \n\
+\t wifiFrequencyBandsSupported : \t "FREQUENCY_BAND_2_4_GHZ" 
+
+APP1 -> SDL : <b>RAI_1(NetworkingCapabilities): \n\
+\t autoJoinWiFiSupported : \t true \n\
+\t canHostWiFiNetwork : \t true \n\
+\t preferredNetworkHost : \t **VEHICLE** \n\
+\t wifiFrequencyBandsSupported : \t "FREQUENCY_BAND_2_4_GHZ" 
+
+APP2 -> SDL : <b>RAI_2(NetworkingCapabilities): \n\
+\t autoJoinWiFiSupported : \t true \n\
+\t canHostWiFiNetwork : \t true \n\
+\t preferredNetworkHost : \t **VEHICLE** \n\
+\t wifiFrequencyBandsSupported : \t "FREQUENCY_BAND_2_4_GHZ" 
+
+
+SDL -> HMI : <b>OnAppRegistered(App1) 
+SDL -> APP1 : <b>RAI(App1) response 
+
+SDL -> SDL : Based on ini file, **RAI_1** and HMI NetworkCapabilities\n\
+SDL decide that **Mobile should be AP** 
+
+SDL -> HMI : <b>OnAppRegistered(App2) 
+SDL -> APP2 : <b>RAI(App2) response 
+
+SDL -> SDL : SDL **postpons** calculation of network configuration for APP2\n\
+until OnSystemRequestCapabilitiesUpdate notification from HMI
+activate SDL
+
+SDL -> HMI:  <b>OnNetworkConfiguration(NetworkConfiguration): \n\
+\t networkHost : \t **MOBILE** \n\
+\t wifiFrequencyBandsSupported : \t "FREQUENCY_BAND_2_4_GHZ" 
+
+SDL -> APP1:  <b>OnNetworkConfiguration(NetworkConfiguration): \n\
+\t networkHost : \t **MOBILE**\n\
+\t wifiFrequencyBandsSupported : \t "FREQUENCY_BAND_2_4_GHZ" 
+
+APP1 -> APP1 : Mobile creates \n\
+WiFi Aceess Point
+
+
+APP1 -> SDL: <b>JoinNetwork requst \n\
+\t ssid : \t DevicerWiFiAP \n\
+\t password : \t 12345678 \n\
+\t securityType : \t WIFI_SECURITY_WPA2 \n\
+\t allowAccessPointToBeShared : \t **true** \n\
+\t accesssPointSupportsInternetAccess : \t true
+
+SDL -> HMI: <b>JoinNetwork request \n\
+\t ssid : \t DeviceWiFiAP \n\
+\t password : \t 12345678 \n\
+\t securityType : \t WIFI_SECURITY_WPA2 \n\
+\t allowAccessPointToBeShared : \t **true** \n\
+\t accesssPointSupportsInternetAccess : \t true
+
+HMI -> HMI : Establish \n\
+WiFiConnection
+
+HMI -> SDL : <b>JoinNetwork response
+SDL -> APP1 : <b>JoinNetwork response
+
+APP1 -> SDL : <b>Start session via WIFI secondary transport
+
+HMI -> SDL:  <b>OnSystemCapabilityUpdate(NetworkConfiguration): \n\
+\t autoJoinWiFiSupported : \t **false** \n\
+\t canHostWiFiNetwork : \t **true**  \n\
+\t preferredNetworkHost : \t **EXTERNAL** \n\
+\t wifiFrequencyBandsSupported : \t "FREQUENCY_BAND_2_4_GHZ" 
+
+deactivate SDL
+ 
+SDL -> SDL : SDL calculate network configuration based on \n\
+lates OnSystemCapabilityUpdate(Networkcapabilities) from HMI, \n\
```

Comment:
Is "lates" intended here?  I'm not sure what this means.

Reply:
Fixed with 5a6ba90aa4a7e438dfd9cc10f87cedc4fa7c5e92